README.md

```diff
@@ -20,109 +26,101 @@ The package Flask-RESTplus has been patched (see `flask_restplus_patched` folder
-  rm -rf migrations/versions/<new migrations>.py
-  invoke app.db.migrate
-  invoke app.db.upgrade
-  invoke app.db.history  # Check that the history matches
-  invoke app.db.heads  # Ensure that there is only one head
-  ```
-* **Ensure that the PR is properly tested**
-  * We require new feature code to be tested via Python tests and simulated REST API tests.  We use PyTest  to ensure that your code is working cohesively and that any new functionality is exercised.
-  * We require new feature code to also be fully compatible with a containerized runtime environment like Docker.
-  ```
-  pytest
-  ```
-* **Ensure that the PR is properly covered**
-  * We require new feature code to be tested (previous item) and that the percentage of the code covered by tests does not decrease.  We use PyTest Coverage and CodeCov.io to ensure that your code is being properly covered by new tests.
-  * To export a HTML report of the current coverage statistics, run the following:
-  ```
-  pytest -s -v --cov=./ --cov-report=html
-  open _coverage/html/index.html
-  ```
-* **Ensure that the PR is properly sanitized**
-  * We require the PR to not include large files (images, database files, etc) without using [GitHub Large File Store (LFS)](https://git-lfs.github.com).
-  ```
-  git lfs install
-  git lfs track "*.png"
-  git add .gitattributes
-  git add image.png
-  git commit -m "Add new image file"
-  git push
-  ```
-  * We also require any sensitive code, configurations, or pre-specified values be omitted, truncated, or redacted.  For example, the file `_db/secrets/py` is not committed into the repository and is ignored by `.gitignore`.
-* **Ensure that the PR is properly reviewed**
-  * After the preceding checks are satisfied, the code is ready for review.  All PRs are required to be reviewed and approved by at least one registered contributor or administrator on the Houston project.
-  * When the PR is created in GitHub, make sure that the repository is specified as `WildMeOrg/houston` and not its original fork.  Further, make sure that the base branch is specified as `develop` and not `master`.
-
-### Pre-commit
-
-It's recommended that you use `pre-commit` to ensure linting procedures are run
-on any commit you make. (See also [pre-commit.com](https://pre-commit.com/)
-
-Reference [pre-commit's installation instructions](https://pre-commit.com/#install) for software installation on your OS/platform. After you have the software installed, run ``pre-commit install`` on the command line. Now every time you commit to this project's code base the linter procedures will automatically run over the changed files.  To run pre-commit on files preemtively from the command line use:
+See [Contributing to Houston](CONTRIBUTING.md)
+
+## Installation
+
+### Using docker-compose (recommended)
+
+#### Setup
 
 ```bash
-  git add .
-  pre-commit run
+git clone --recurse-submodules https://github.com/WildMeOrg/houston.git
+cd houston
+# build the frontend
```

Comment:
Some of these components are already in scripts/install.sh, Rather than a list of commands in the README, that script should be updated to do all that's required and referenced from here

Reply:
In the context of setting up for docker... The `install.sh` script is more tailored to local host system development. Part of the direction with docker usage is to minimally require the dependencies on the host system. So with this docker setup, you shouldn't need to install anything but docker (and python for pre-commit).